dashboard/modules/log/tests/test_log.py

```diff
@@ -136,7 +136,13 @@ def write_log(s):
 
 @pytest.mark.parametrize(
     "test_file",
-    ["test.log", "test#1234.log"],
+    [
```

Comment:
can we test with actual binaray file? I think memray can generate a binary file (it doesn't have to be in the test, but you can do it manually)

Reply:
sure - although I feel the actual content of the file probably doesn't matter here. It will just be binary content interpreted as text on the frontend?

Reply:
I think I'd like to make sure download works. I.e., run memray, download, and make sure I can see the profiling file.